command/sidechain/registration/params.go

```diff
@@ -53,15 +51,24 @@ func (rr registerResult) GetOutput() string {
 
 	var vals []string
 
-	buffer.WriteString("\n[REGISTRATION]\n")
+	buffer.WriteString("\n[SUCCESSFUL REGISTRATION]\n")
 
-	vals = make([]string, 0, 3)
-	vals = append(vals, fmt.Sprintf("Validator Address|%s", rr.validatorAddress))
-	vals = append(vals, fmt.Sprintf("Staking Result|%s", rr.stakeResult))
-	vals = append(vals, fmt.Sprintf("Amount Staked|%v", rr.amount))
+	vals = make([]string, 0, 1)
+	vals = append(vals, fmt.Sprintf("EVM Address|%s", rr.validatorAddress))
 
 	buffer.WriteString(helper.FormatKV(vals))
 	buffer.WriteString("\n")
 
+	if rr.stakeResult != "" {
+		buffer.WriteString("\n[SELF STAKE]\n")
+
+		vals = make([]string, 0, 2)
+		vals = append(vals, fmt.Sprintf("Staking Result|%s", rr.stakeResult))
+		vals = append(vals, fmt.Sprintf("Amount Staked|%v", rr.amount))
+
+		buffer.WriteString(helper.FormatKV(vals))
+		buffer.WriteString("\n")
+	}
```

Comment:
Otherwise, add error message that the register is successful but the stake failed + the actual error so people now what happened

Reply:
@R-Santev, it currently works almost the way you described, but the actual error is not added in the failed message. However, the code for this is in the `register_validator.go`.